packages/@aws-cdk/integ-runner/lib/workers/common.ts

```diff
@@ -152,9 +152,19 @@ export interface IntegTestOptions {
    * Whether to enable verbose logging
    *
    * @default false
+   * @deprecated - use `verbosity` instead
```

Comment:
Since this is an experimental API, should I just remove the prop?

Reply:
What do you think @corymhall ?

Reply:
I would be in favor of removing it.

Reply:
yeah we can just remove it.